### ISSUE TITLE
Spread of any object into jsx causes react/prop-types to stop working (errors not detected)

### ISSUE THREAD

**Issue by kitfit-dave** [yannickcr/eslint-plugin-react#1952]
Here is a minimal example:

```
import React from 'react'

class MyComponent extends React.Component {
	render() {
		// should be "'someProp' is missing in props validation (react/prop-types)" but is not
		this.props.someProp()

		return <div {...{}} />
	}
}

export default MyComponent
```

remove the spread ```{...{}}``` and everything is fine

**Comment by alexzherdev** [comment]
This was filed as #1601 for `no-unused-prop-types` and there's an unfinished PR in #1646.
If #1946 is merged, the fix will go into that common helper and apply for both rules.

**Comment by ljharb** [comment]
In that case, I'll close this now.
